libraries/AP_Motors/AP_MotorsPulsing_Heli.cpp

```diff
@@ -0,0 +1,220 @@
+#include <AP_HAL/AP_HAL.h>
+#include <AP_Math/AP_Math.h>
+#include "AP_MotorsPulsing_Heli.h"
+#include <GCS_MAVLink/GCS.h>
+#include <SRV_Channel/SRV_Channel.h>
+
+extern const AP_HAL::HAL& hal;
+
```

Comment:
tangent here: I notices that the descriptions/values/etc. do not show up in either of the ground station softwares. The internet seems to think that after everything gets in, they'll automagically start showing up as part of the real ardupilot build.